app/controllers/api/v1/participants_controller.rb

```diff
@@ -2,7 +2,42 @@ module Api::V1
   class ParticipantsController < ApplicationController
 
     def index
-      render json: Participant.all
+      render json: Participant.accepted
+    end
+
+    def create
+      participant = Participant.new(participant_params)
+
+      if participant.save
+        render json: participant
+      else
+        render json: { error: participant.errors.full_messages }
+      end
+    end
+
+    def update
+      unless Participant.statuses.keys.include?(params[:status])
```

Comment:
nice constraint! 

I think we could make this more readable doing something like this

```
  def update
     ....
     return invalid_status unless status_param_valid?
     ....
  end

  private

  def invalid_status
    render json: { message: "Invalid status" }
  end

  def status_param_valid?
    Participant.statuses.keys.include?(params[:status])
  end
```

just saying! :+1:  @topdev531